node-and-npm/node-windows.md

```diff
@@ -0,0 +1,41 @@
+## Install Node And npm On Windows
+
+### Install Node
+
+1. Go to the [Node Download for Windows page](https://nodejs.org/en) and select the latest version to download.
+
+2. Node will begin to download on your computer. You will see the download's progress in the lower left-hand corner of your screen. 
+
+3. Once the download is complete, click the file in the lower left-hand corner to open the installer. 
+
+4. Click **Next** to work through the installer using all default settings. When you reach the end of the steps, select **Install**. 
+
+5. Once complete, verify your installation by opening the **Command Prompt** \(search for "cmd" in your computer's **Start** menu\).
+
+    ![Open Windows Command Prompt](../.gitbook/assets/cmd.png)
+
+6. In the **Command Prompt**, type the following command and press `Enter`. You will always press `Enter` after typing a command in the **Command Prompt** unless otherwise instructed.
+
+```bash
+node --version
+```
+
+7. If the output contains a number as shown below, **Node** installed successfully.
+
+![Node version successful output](../.gitbook/assets/node-version-win.png)
+
+### Install npm
+
+1. To install or update to the latest version of **npm**, in the **Command Prompt**, type the following command.
+
+```bash
+npm install npm -g
+```
+2. Once the process completes, verify your installation by typing the following command.
+
+```bash
+npm --version
+```
+3. If the output contains a number as shown below, **npm** installed successfully.
```

Comment:
Let's add the instructions
> Close the command prompt window

When they install Git, they will need to reopen command prompt (can't use the same prompt since Git isn't on the PATH) so it's easier to instruct to close than to get confused on which terminal they have open.

Might consider that with the macOs instructions too....